syftbox/client/core.py

```diff
@@ -332,11 +283,11 @@ def run_migration(config: SyftClientConfig, migrate_datasite=True):
             shutil.rmtree(str(old_sync_state.parent))
 
 
-def run_client(
+def run_syftbox(
```

Comment:
maybe somehow indicate that this is the client side? or do you think that is clear enough from the path?